stable/redmine/values.yaml

```diff
@@ -75,12 +75,10 @@ redmineLanguage: en
 #   - name: NAMI_DEBUG
 #     value: --log-level trace
 
-## Database configuration. Please note that only one of the following databases should be selected.
+## Database type. Allowed values: "mariadb" and "postgresql"
 ## ref: https://github.com/bitnami/bitnami-docker-redmine#run-the-application-using-postgresql-database
 ##
-databaseType:
-  mariadb: true
-  postgresql: false
+databaseType: mariadb
```

Comment:
Does this change warrant a major version bump? At any rate we should document this change in the README

Reply:
If we are strict, yes, it requires a major version bump. We can:
- a) Bum the major version (removing the support to the old way to configure ingress)
- b) Document the change in the README.md
- c) Both of them

Reply:
What do you prefer?

Reply:
I would prefer the major version bump